README.md

```diff
@@ -510,8 +510,8 @@ point := struct {
 ## Pointers
 ```go
 p := Vertex{1, 2}  // p is a Vertex
-q := &p            // q is a pointer to a Vertex
-r := &Vertex{1, 2} // r is also a pointer to a Vertex
+q := &p            // q is a pointer to "p" Vertex
+r := &Vertex{1, 2} // r is also a pointer to another Vertex
```

Comment:
i reckon `a pointer to a Vertex` should be read as a whole here.

q is `a pointer to a Vertex`
r is also `a pointer to a Vertex`

Reply:
For my understanding, `q` is a pointer to `p` in memory, while `r` points to a new in-memory object.
Right?

Reply:
p is a value of vertex, q is a pointer to this value.

Reply:
Understood; thanks for the explanation.